service/topology_state_machine.cc

```diff
@@ -147,7 +147,7 @@ static std::unordered_map<topology::transition_state, sstring> transition_state_
     {topology::transition_state::write_both_read_old, "write both read old"},
     {topology::transition_state::write_both_read_new, "write both read new"},
     {topology::transition_state::tablet_migration, "tablet migration"},
-    {topology::transition_state::tablet_split_finalization, "tablet split finalization"},
+    {topology::transition_state::tablet_resize_finalization, "tablet resize finalization"},
```

Comment:
Is this a compatibility problem?

Reply:
good catch, we do reverse lookup using string to transition id, when loading topology state. I can add tablet_merge_finalization and do this:
```
            case topology::transition_state::tablet_split_finalization:
            case topology::transition_state::tablet_merge_finalization:
                co_await handle_tablet_resize_finalization(std::move(guard));
```